gui/default/index.html

```diff
@@ -548,7 +548,7 @@ <h4 class="panel-title">
                     <button ng-if="folder.paused" type="button" class="btn btn-sm btn-default" ng-click="setFolderPause(folder.id, false)">
                       <span class="fas fa-play"></span>&nbsp;<span translate>Resume</span>
                     </button>
-                    <button type="button" class="btn btn-default btn-sm" ng-click="restoreVersions.show(folder.id)" ng-if="folder.versioning.type">
+                    <button type="button" class="btn btn-default btn-sm" ng-click="restoreVersions.show(folder.id)" ng-if="folder.versioning.type != 'external'">
```

Comment:
Doesn't this make the button show up when versioning is disabled all together?

Reply:
Of course it does… :flushed:

Should be fixed now.